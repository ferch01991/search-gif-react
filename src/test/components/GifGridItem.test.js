import React from 'react'

import { shallow } from 'enzyme'
import { GifGridItem } from '../../GifGridItem'

describe('Prueba para el componente GifGridItem', () => {

    const title = 'Un ttitulo';
    const url   = 'http://localhost/algo.jpg';

    test('should show component correctly', () => {
        const wrapper = shallow( <GifGridItem title = {title} url={url} /> )
        expect( wrapper ).toMatchSnapshot();
    })
    
    
})