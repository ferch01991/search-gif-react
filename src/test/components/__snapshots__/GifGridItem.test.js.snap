// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba para el componente GifGridItem should show component correctly 1`] = `
<div
  className="card animate__animated animate__bounce"
>
  <img
    alt="Un ttitulo"
    src="http://localhost/algo.jpg"
  />
  <p>
    Un ttitulo
  </p>
</div>
`;
